server/Mistakes.Journal.Api/Api/Mistakes/Mappers/MistakeMapper.cs

```diff
@@ -15,7 +16,30 @@ public static MistakeWebModel ToWebModel(this Mistake mistake)
                 Name = mistake.Name,
                 Goal = mistake.Goal,
                 Priority = mistake.Priority,
+                Labels = mistake.MistakeLabels.Select(ml => ml.Label.ToWebModel()).ToList(),
                 Tips = mistake.Tips.Select(t => t.Content).ToList(),
+                RepetitionDates = mistake.Repetitions.Select(r => r.ToWebModel()).ToList(),
+            };
+        }
+
+        public static Mistake ToEntity(this NewMistakeWebModel newMistake)
+        {
+            return new Mistake(
+                newMistake.Name,
+                newMistake.Goal,
+                newMistake.Priority.GetValueOrDefault());
+        }
+
+        public static NewMistakeWebModel ToNewMistakeWebModel(this Mistake mistake)
+        {
+            return new NewMistakeWebModel
+            {
+                Name = mistake.Name,
+                Goal = mistake.Goal,
+                Priority = mistake.Priority,
+                Tips = mistake.Tips.Select(t => t.Content).ToList(),
+                Labels = mistake.MistakeLabels.Select(ml => ml.LabelId).ToList(),
+                AddDateTime = mistake.Repetitions.First().DateTime,
```

Comment:
What is `AddDateTime`? I think `CreatedAt` is better name for creation date :D Plus, I see that you only return this in NewMistakeWebModel. I think `CreatedAt` would be  useful in MistakeWebmodel.

Reply:
Are you also sure that first repetition is the first one? Are they ordered by default by date? I am not sure that from business point of view 1st repetition == CreatedAt, but I might be wrong, so therefore question arises :)

Reply:
I assumed so